_portal-quickstarts

```diff
@@ -1 +1 @@
-Subproject commit caf178764214cd14a6c93b65308343945f755cd1
+Subproject commit c69f4643a0ab20d6777ebb30a00bffec726d890a
```

Comment:
Again, I'm not sure this is right. You'll need to find out which one correlates to latest product release.